src/operator/OrgOverlay.tsx

```diff
@@ -0,0 +1,242 @@
+
+import {OverlayContext} from 'src/operator/context/overlay'
+import LimitsInput from 'src/operator/LimitsInput'
+import {fromDisplayLimits} from 'src/operator/utils'
+
+const OrgOverlay: FC = () => {
+  const {
+    limits,
+    limitsStatus,
+    handleUpdateLimits,
+    organization,
+    orgStatus,
+    setLimits,
+    updateLimitStatus,
+  } = useContext(OverlayContext)
+
+  const {orgID} = useParams()
+  const history = useHistory()
+
+  const setInputs = limits => {
+    setLimits(limits)
+  }
+
+  const updateLimits = async () => {
+    try {
+      const backendLimits = fromDisplayLimits(limits)
+      await handleUpdateLimits(backendLimits)
+      history.goBack()
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
+  // TODO(ariel): fix the background issue by this being on a separate page
+  return (
+    <Overlay
+      visible={true}
+      renderMaskElement={() => (
+        <Overlay.Mask gradient={Gradients.PowerStone} style={{opacity: 0.5}} />
+      )}
+      testID="org-overlay"
+      transitionDuration={0}
+    >
+      <Overlay.Container maxWidth={1000}>
+        <Overlay.Header
+          title={orgID}
+          style={{color: '#FFFFFF'}}
+          onDismiss={() => history.goBack()}
+        />
+        <SpinnerContainer
+          loading={orgStatus}
+          spinnerComponent={<TechnoSpinner diameterPixels={100} />}
+        >
+          <Overlay.Body>
+            <Grid>
+              <Grid.Row>
+                <Grid.Column widthMD={4}>
+                  <label>Organization Name</label>
+                  <p>{organization?.name ?? ''}</p>
+                </Grid.Column>
+                <Grid.Column widthMD={4}>
+                  <label>Account Type</label>
+                  <p>{organization?.account?.type ?? ''}</p>
+                </Grid.Column>
+                <Grid.Column widthMD={4}>
+                  <LinkButton
+                    color={ComponentColor.Secondary}
+                    size={ComponentSize.Small}
+                    shape={ButtonShape.Default}
+                    style={{padding: '7px 10px'}}
+                    testID="usage-button"
+                    text="View Usage Dashboard"
+                    target={LinkTarget.Blank}
+                    href={`https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/0649b03029c49000?vars%5Borgid%5D=${orgID}`}
+                  />
+                </Grid.Column>
+              </Grid.Row>
+              <SpinnerContainer
+                loading={limitsStatus}
+                spinnerComponent={<TechnoSpinner diameterPixels={100} />}
+                testID="limits-spinner-container"
+              >
+                <Grid.Row>
+                  <h4>Limits</h4>
+                  <Grid.Column widthMD={3}>
+                    <Form.Label label="Read (KBs)" testID="read-kbs" />
+                    <LimitsInput
+                      type={InputType.Number}
+                      name="rate.readKBs"
```

Comment:
As mentioned above, I don't believe that these `name` props will work without using lodash's `get` and `set` in `LimitsInput`.